oci/oci.go

```diff
@@ -593,16 +594,21 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 	defer c.opLock.Unlock()
 	out, err := exec.Command(r.Path(c), "state", c.id).CombinedOutput()
 	if err != nil {
-		// there are many code paths that could lead to have a bad state in the
-		// underlying runtime.
-		// On any error like a container went away or we rebooted and containers
-		// went away we do not error out stopping kubernetes to recover.
-		// We always populate the fields below so kube can restart/reschedule
-		// containers failing.
-		c.state.Status = ContainerStateStopped
-		c.state.Finished = time.Now()
-		c.state.ExitCode = 255
-		return nil
+		switch err.(type) {
```

Comment:
I don't think that we should be adding code for checking the runtime binary here. If anything just add a exec.LookPath when cri-o starts up.

Reply:
We do have lookpath in cri-o start afaict also

Reply:
fixed, moved into kpod/common.go in similar fashion